### PR TITLE
Update to latest Gateway service APIs

### PR DESCRIPTION
**Pull request by JoeCap08055**
# Purpose
The goal of this PR is to update the SAT backend with all of the latest APIs published by the various Gateway services

### Change summary
- [x] Update API & calls for `account-service`
- [x] Update API & calls for `content-publishing-service`
- [x] Update API & calls for `content-watcher-service`
- [x] Update API & calls for `graph-service`

## Steps to Verify
- [ ] test all functions in the SAT

### CODE REVIEW

File: backend/scripts/get-openapi-specs.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+}
+
+cd openapi-specs
+
+# Download the swagger.json files from the microservices repositories
+download_oopenapi_specs "AmplicaLabs" "account-service" "main" "swagger.json" "account-service.json"
+download_oopenapi_specs "AmplicaLabs" "content-watcher-service" "main" "content-announcement.openapi.json" "content-announcement.openapi.json"
+download_oopenapi_specs "AmplicaLabs" "content-publishing-service" "main" "swagger.json" "content-publishing-service.json"
+download_oopenapi_specs "AmplicaLabs" "content-watcher-service" "main" "swagger.json" "content-watcher-service.json"
+download_oopenapi_specs "AmplicaLabs" "graph-service" "main" "swagger.json" "graph-service.json"
```
Comment (JoeCap08055) [inline backend/scripts/get-openapi-specs.sh:32]: nit: use a variable for the org to make it easier to change when we rename the company again 😆 

```suggestion
ORG_NAME="AmplicaLabs"
BRANCH="main"
download_oopenapi_specs "${ORG_NAME}" "account-service" "${BRANCH}" "swagger.json" "account-service.json"
download_oopenapi_specs "${ORG_NAME}" "content-watcher-service" "${BRANCH}" "content-announcement.openapi.json" "content-announcement.openapi.json"
download_oopenapi_specs "${ORG_NAME}" "content-publishing-service" "${BRANCH}" "swagger.json" "content-publishing-service.json"
download_oopenapi_specs "${ORG_NAME}" "content-watcher-service" "${BRANCH}" "swagger.json" "content-watcher-service.json"
download_oopenapi_specs "${ORG_NAME}" "graph-service" "${BRANCH}" "swagger.json" "graph-service.json"
```

File: backend/controllers/ProfilesController.ts [hunk 2]
```diff
@@ -21,10 +21,8 @@ export class ProfilesController extends BaseController {
       fromId: _req.params.msaId,
     };
     // Use account service to get display handle
-    const { displayHandle } = await AccountService.getInstance().then((service) =>
-      service.getAccount(_req.params.msaId)
-    );
-    Profile.displayHandle = displayHandle;
+    const { handle } = await AccountService.getInstance().then((service) => service.getAccount(_req.params.msaId));
+    Profile.displayHandle = handle?.displayHandle;
```
Comment (mattheworris) [inline backend/controllers/ProfilesController.ts:25]:
```suggestion
    Profile.handle = handle;
```

**Review by claireolmstead — APPROVED** [review]
LGTM!
- Pair reviewed and tested together

🚢

File: backend/services/AccountService.ts [hunk 3]
```diff
@@ -111,8 +112,8 @@ export class AccountService {
           accessToken: createAuthToken(accountData.accountId),
           expires: Date.now() + 24 * 60 * 60 * 1_000,
           referenceId: referenceId,
-          msaId: parseInt(accountData.msaId),
-          displayHandle: accountData.displayHandle,
+          msaId: accountData.msaId,
+          handle: { displayHandle: accountData.displayHandle },
```
Comment (mattheworris) [inline backend/services/AccountService.ts:116]:
```suggestion
          handle: accountData.handle,
```

**Pull request merged**